lib/icingadb/icingadb-objects.cpp

```diff
@@ -1626,7 +1621,7 @@ void IcingaDB::SendStateChange(const ConfigObject::Ptr& object, const CheckResul
 
 	tie(host, service) = GetHostService(checkable);
 
-	SendStatusUpdate(checkable);
+	UpdateState(checkable, true);
```

Comment:
Previously, this only sent the runtime update which is fine as the corresponding `icinga:*:state` was updated by `OnNewCheckResult`/`NewCheckResultHandler` before. In the meantime, I figured out that `icinga:*:state` is already updated multiple times (but that shouldn't serve as an excuse to add another pointless update).